config/config_test.go

```diff
@@ -0,0 +1,44 @@
+package config
+
+import (
+	"os"
+	"testing"
+	"time"
+
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+func TestConfig(t *testing.T) {
+	Convey("Given an environment with no environment variables set", t, func() {
+		os.Clearenv()
+		cfg, err := Get()
+
+		Convey("When the config values are retrieved", func() {
+
+			Convey("Then there should be no error returned", func() {
+				So(err, ShouldBeNil)
+			})
+
+			Convey("Then the values should be set to the expected defaults", func() {
+				So(cfg.APIRouterURL, ShouldEqual, "http://localhost:23200/v1")
+				So(cfg.BatchMaxWorkers, ShouldEqual, 100)
+				So(cfg.BatchSizeLimit, ShouldEqual, 1000)
+				So(cfg.BindAddr, ShouldEqual, ":20001")
+				So(cfg.DownloadServiceURL, ShouldEqual, "http://localhost:23600")
+				So(cfg.EnableDatasetPreview, ShouldBeFalse)
+				So(cfg.EnableProfiler, ShouldBeFalse)
+				So(cfg.GracefulShutdownTimeout, ShouldEqual, 5*time.Second)
+				So(cfg.HealthCheckCriticalTimeout, ShouldEqual, 90*time.Second)
+				So(cfg.HealthCheckInterval, ShouldEqual, 30*time.Second)
+				So(cfg.MaxDatasetOptions, ShouldEqual, 200)
+				So(cfg.RendererURL, ShouldEqual, "http://localhost:20010")
+			})
+
+			Convey("Then a second call to config should return the same config", func() {
```

Comment:
This convey is meant to check that we don't create a new config object on a second call to the `Get()` function. I believe this is copied from other repo but this one does not have a check for that: it always creates a new Config. Please add the check to the implementation like [this](https://github.com/ONSdigital/dp-dataset-api/blob/develop/config/config.go#L67-L69)